documentation/sphinx/source/backups.rst

```diff
@@ -244,6 +244,9 @@ The ``start`` subcommand is used to start a backup.  If there is already a backu
 ``-s <DURATION>`` or ``--snapshot_interval <DURATION>``  
   Specifies the duration, in seconds, of the inconsistent snapshots written to the backup in continuous mode.  The default is 864000 which is 10 days.
 
+``--init_snapshot_interval <DURATION>``  
```

Comment:
This is a nitpick, but.. I would strongly prefer if this argument were called `--initial_snapshot_interval` because it is a user-facing feature and "init" means something different than "initial" so initial is more clear, especially since whether or not to even "init" as in start a snapshot is or will be configurable as there is a use case for a backup without a snapshot.

Reply:
No problem, done.